rocketmq-amqp/src/main/java/org/apache/rocketmq/amqp/server/AMQPServer.java

```diff
@@ -0,0 +1,28 @@
+package org.apache.rocketmq.amqp.server;
+
+import java.io.IOException;
+import java.net.ServerSocket;
+import java.net.Socket;
+
+public class AMQPServer {
+
+    private int amqpPort = 5672;
+
+    public void execute() throws IOException {
+        runServer(amqpPort);
+    }
+
+    public void runServer(int port) throws IOException {
+        ServerSocket serverSocket = null;
+        Socket socket = null;
+
+        serverSocket = new ServerSocket(port);
+        while (true)
+        {
+            socket = serverSocket.accept();
+            //new thread for a client
+            new ClientThread(socket).start();
+        }
```

Comment:
One socket one thread? If thousands of sockets come in, the server will run into a heavy burden.

Reply:
Yes , I have realized it later. So I'm now working to connect using channel. I think it will solve the problem. Am I right?